assemblies/assembly-enabling-authentication.adoc

```diff
@@ -34,7 +34,7 @@ For each provider that you want to use, follow the dedicated procedure to comple
 . Set up the shared secret that the authentication provider and {product} require to communicate.
```

Comment:
The assembly is missing the `:context: variable-name`. It needs to be in every assebmly ([link for more information](https://redhat-documentation.github.io/modular-docs/#reusing-modules))

Because I see that in this assembly there are nested assemblies, this: [section](https://redhat-documentation.github.io/modular-docs/#nesting-assemblies) needs to be kept in mind.